### PR TITLE
🚨Fixed all the linting warnings

### LINKED ISSUE #4: Fix eslint errors
`npm run lint` and fix the errors & warnings

### PR DESCRIPTION
**Pull request by kevinmeyvaert**
You're welcome.

This PR closes #4

### CODE REVIEW

**Comment by easybird** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-4-fix-linterrors

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by easybird** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-4-fix-linterrors

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

File: utils/colors.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const COLORS = {
```
Comment (thibmaek) [inline utils/colors.js]: Typed as uppercase `COLORS` but named import as `Colors` 😔

**Comment by easybird** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-4-fix-linterrors

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by easybird** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-4-fix-linterrors

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Pull request merged**